src/catppuccin-latte-blue.json

```diff
@@ -26,22 +26,22 @@
         "primary_dark_660": -2301720,
         "primary_dark_360": -8618093,
         "uikit_settings_item_text_color_light": -6512464,
+        "uikit_settings_item_text_color_dark": -11776151,
         "primary_dark_300": -11776151,
-        "uikit_btn_bg_color_selector_red": -8618093,
+        "uikit_btn_bg_color_selector_red": -3010759,
         "uikit_btn_bg_color_selector_green": -8618093,
         "link": -14784779,
         "link_500": -14784779,
         "primary_660": -14784779,
+        "white_500": -11776151,
         "primary_630": -1644049,
         "uikit_btn_compound_color_selector_dark": -14784779,
-        "status_grey_200": -2301720,
-        "brand_500_alpha_20": 1042179829,
+        "status_grey_200": -11776151,
```

Comment:
I've gone through and reviewed all the changes again (pretty slow but - go to https://www.rapidtables.com/convert/number/decimal-to-hex.html, put in the number, get the hex output, compare to Catppuccin output) and this is the only one that looks incorrect. On the left, `status_grey_200` was mantle, but on the right, it is now text. Seeing as this is latte, I can see how this might be... but intriguingly this is the only color that seems to have changed to the complete opposite shade in this file.